src/graph/graphcommandexecutor.pas

```diff
@@ -99,7 +99,7 @@ procedure TGraphCommandExecutor.ShowDialog(ST: TCustomGraphCommand;
 procedure TGraphCommandExecutor.ShowMarksAsDates(var AText: String;
   AMark: Double);
 begin
-  AText := DateToStr(AMark);
+  AText := FormatDateTime('dd/mm/yyyy', AMark);
```

Comment:
No, this will always format as DDMMYYYY which is not desired. If a user has a different locale they should be allowed to view data in their own specified locale. No matter the content of data.

Reply:
I agree. 
As written, DateToStr does not respect the default on my Mac.
My default short date is YYYY-MM-DD or something like that and DateToStr yields 'dd-mm-yy'. I'll see if I can get the desired short date out using the DateToStr options.

Reply:
See pull request re: localize formats for MacOS. I've put that in as a separate PR as it has an impact on how dates are shown across the app. But note bug in Lazarus or FPC fails to pick up yyyy, always treating year as yy.